AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientTests.swift

```diff
@@ -793,6 +795,37 @@ class AWSMobileClientTests: XCTestCase {
         
     }
     
+    func testGetIdentityId() {
+        XCTAssertNil(AWSMobileClient.sharedInstance().identityId, "Identity Id should be nil after initialize.")
+
+        let identityIdExpectation = expectation(description: "IdentityID should be retrieved successfully")
```

Comment:
Nit: you're fulfilling the expectation upon any invocation of the block, not only successful retrieval. Please make the name & description conform to what you're actually using it for.

Reply:
Renamed the expectation.